app/services/weets.js

```diff
@@ -32,8 +32,11 @@ exports.createWeet = (userId, content) =>
     });
 
 exports.getPaginatedWeets = async ({ size = defaultPaginationSize, page = defaultPage }) => {
-  const { count, rows } = await weetModel.findAndCountAll({ limit: size, offset: page * size });
-  const weets = rows.map(weet => weet.toJSON());
+  const { count, rows: weets } = await weetModel.findAndCountAll({
```

Comment:
Which is the criterium to send these emails?

Reply:
as the card did not say, we are currently sending the congratulations email to all the users at 00:00 every day

Reply:
we can look for the top N (environment variable) Users that created the most weets in a day